server.go

```diff
@@ -396,10 +408,24 @@ func (s *Server) processStreamingRPC(t transport.ServerTransport, stream *transp
 			ss.statusDesc = appErr.Error()
 		}
 	}
+	if ss.tracing {
+		ss.mu.Lock()
+		if ss.statusCode != codes.OK {
+			ss.traceInfo.tr.LazyLog(stringer(ss.statusDesc), true)
+			ss.traceInfo.tr.SetError()
+		} else {
+			ss.traceInfo.tr.LazyLog(stringer("OK"), true)
```

Comment:
We need stringer for LazyLog(stringer(statusDesc), true) so that we can set its sensitivity and ensure statusDesc is not interpreted as a format string.  While we could write LazyPrintf("OK"), I don't see why we should, given that LazyLog(stringer("OK"), false) does what we want.

But personally I'd prefer to have a Trace.Log(string) method to sidestep this whole issue.  I can send that in a separate CL if you're open to it.